### DISCUSSION TITLE
Not really sure what 'Upscaled' option does

### DISCUSSION THREAD

**Discussion by gameboin** [Coyote-A/ultimate-upscale-for-automatic1111#49] [Q&A] [answered]
First of all I've been using this upscaler for awhile and love it! 
But what does the Upscaled option do?
I know in FAQ it says:
Upscaled - Enabled by default. Saving image from Redraw

I'm not sure what that means, but I do know that I've tested upscaled option both on and off while keeping all the other settings the same, and my result is both images are nearly identical.

**Comment by Jopezzia** [comment, marked as answer]
This option is responsible for saving the image (without seams fix) to the img2img folder. If you turn it off, the image will not be saved. Same with seams fix
